### DISCUSSION TITLE
500 Server Error when updating to 0.12.0

### DISCUSSION THREAD

**Discussion by mickehallgren** [blakeblackshear/frigate#6282] [General]
Hi!
I'm running Home Assistant on a NUC with Home Assistant OS installation.
When I try to update I get this error:
Can't install ghcr.io/blakeblackshear/frigate:0.12.0: 500 Server Error for http+docker://localhost/v1.42/images/create?tag=0.12.0&fromImage=ghcr.io%2Fblakeblackshear%2Ffrigate&platform=linux%2Famd64: Internal Server Error ("Head "https://ghcr.io/v2/blakeblackshear/frigate/manifests/0.12.0": received unexpected HTTP status: 503 Service Unavailable")

Is anyone that can come up with a suggestion on what is wrong? Thanks!

**Comment by blakeblackshear** [comment]
I would just try again. It may have just been a service outage in ghcr.

**Reply by mickehallgren** [reply]
Thanks a lot! I did try like 20 times, but now it's working!
